pkgs/tools/graphics/pdf-sign/default.nix

```diff
@@ -1,17 +1,19 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, substituteAll
 
 , python3
+
 , ghostscript
-, coreutils
 , pdftk
 , poppler_utils
+
+, makeWrapper
```

Comment:
```suggestion
, makeBinaryWrapper
```

Reply:
I don't see why it's better to use this. Is this faster?

Reply:
I guess it would be faster, but that's not the reason why I always advise people to use it.

The regular `makeWrapper` might cause some issue when running on Darwin, therefore, it's better to take the habit and use `makeBinaryWrapper` instead by default.

Reply:
That's good to know :) thx